src/components/legend/defaults.js

```diff
@@ -17,7 +17,7 @@ var basePlotLayoutAttributes = require('../../plots/layout_attributes');
 var helpers = require('./helpers');
 
 
-module.exports = function legendDefaults(layoutIn, layoutOut, fullData) {
+module.exports = function legendDefaults(layoutIn, layoutOut, fullData, fullLayout) {
```

Comment:
Let's make the 4th argument an option
```js
module.exports = function legendDefaults(layoutIn, layoutOut, fullData, opts) {
  if(!opts) opts = {};
```
and then pass desired options like this:
```js
legend.supplyLayoutDefaults(mockLayoutIn, mockLayoutOut, gd._fullData, {
  paper_bgcolor: fullLayout.paper_bgcolor,
  font: fullLayout.font
});
```

Reply:
Following @etpinard's https://github.com/plotly/plotly.js/pull/4620#discussion_r390317487 and yours, I realized that I can simply set the `font`, `papger_bgcolor` and `title.font` in the `mockLayoutIn` and not have to pass the `fullLayout` object to `legend.supplyLayoutDefaults`. `fullLayout` is only needed to figure out the default values in case they are missing.

Reply:
Done in https://github.com/plotly/plotly.js/commit/9ea8bfdab29495a2eaf96884a36dff71a38aff67

Reply:
Resolved in https://github.com/plotly/plotly.js/pull/4620/commits/9ea8bfdab29495a2eaf96884a36dff71a38aff67.